runtime/src/main/java/com/dylibso/chicory/runtime/GlobalInstance.java

```diff
@@ -1,13 +1,21 @@
 package com.dylibso.chicory.runtime;
 
+import com.dylibso.chicory.wasm.types.MutabilityType;
 import com.dylibso.chicory.wasm.types.Value;
 
 public class GlobalInstance {
     private Value value;
     private Instance instance;
+    private MutabilityType mutabilityType;
```

Comment:
make it final and remove the setter?

Reply:
Reason I didn't do it is for consistency with the rest of the class, I would have suggested to do it all at once later, turning the entire class immutable. But I can do it for this field if you prefer

Reply:
thanks